NordigenSwiftDemo/NordigenSwiftDemo/Scenes/Main/ContentViewModel.swift

```diff
@@ -98,17 +100,29 @@ final class ContentViewModel: ObservableObject {
         }
     }
 
-    func connectToSandboxInstitution() {
+    func connectToInstitution(id: String? = nil) {
         isConnectingToSandbox = true
 
-        let sandboxInstitutionId = "SANDBOXFINANCE_SFIN0000"
+        let institutionId = id ?? "SANDBOXFINANCE_SFIN0000"
 
         Task {
             do {
-                let requisition = try await nordigenClient.RequisitionsAPI.new( .init(institutionId: sandboxInstitutionId, redirect: "NordigenSwiftDemo://BankAuthenticationRedirect?id=\(sandboxInstitutionId)"))
+                let payload = EndUserAgreementWriteRequest(
```

Comment:
Creating a custom EndUserAgreement should not be hardcoded for the default use case. I'd suggest to create a separate flow to do so

Reply:
I have removed the custom end user agreement from the default use case in the latest commit.